frame/collective/src/lib.rs

```diff
@@ -181,7 +181,8 @@ pub mod pallet {
 		type Proposal: Parameter
 			+ Dispatchable<Origin = <Self as Config<I>>::Origin, PostInfo = PostDispatchInfo>
 			+ From<frame_system::Call<Self>>
-			+ GetDispatchInfo;
+			+ GetDispatchInfo
+			+ MaxEncodedLen;
```

Comment:
Need a pointer here. This breaks any instantiation that do `type Proposal = Call;` (which is all of them?) and I'm sort of stuck here. :/

Reply:
Haha, you picked one of the hardest one to work on :). This is almost exactly the same issue that we've faced in https://github.com/paritytech/substrate/pull/10202#discussion_r745344182, and in short, the way we have to fix this is to make it so that the `Call` enum constructed by `construct_runtime` implements `MaxEncodedLen`, which is as difficult as it sounds.

Reply:
Can we not `Box<>` it?

Reply:
> Can we not Box<> it?

You mean do like `type Proposal = Box<Call>;`?

Reply:
Ah NVM, that trick only works on function arguments.  
I have the same problem right now in the scheduler where calls are stored...

Reply:
The suggestion from chat was to store the encoded version of the call.  
I will try that in the scheduler to see how it looks like.